client/playwright.config.ts

```diff
@@ -4,11 +4,10 @@ export default defineConfig({
   testDir: './e2e',
   timeout: 30 * 1000,
   expect: {
-    timeout: 5000,
+    timeout: 10 * 1000,
   },
-  fullyParallel: true,
   forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
+  workers: 1,
```

Comment:
couldn't get the parallized workers setup working which I _think_ is a function of the fact that we're doing logins at the worker level. We should be able to bump this up as we get more worker accounts.

Reply:
Yeah I thought it'd be fine to use multiple workers because of this fact as well, but in practice, think something else is going wrong. It does seem that just using the one worker is making the tests pass every time though, so I opted to just set the one worker up for now until I have the chance to look further into it in a followup.